src/ppspp_options.erl

```diff
@@ -237,8 +248,8 @@ get_content_integrity_check_method(Options) ->
 %% @end
 -spec get_merkle_hash_tree_function(options()) -> any().
 
-get_merkle_hash_tree_function(Options) ->
-    get(merkle_hash_tree_function, Options).
+get_merkle_hash_tree_function(#{merkle_hash_tree_function := HashFunc} = _Options) ->
+    HashFunc.
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %% @doc get_minimum_version/1 returns the lowest accepted PPSP version for the swarm
```

Comment:
Line 255 is too long: %% @doc get_minimum_version/1 returns the lowest accepted PPSP version for the swarm.